lib/fluent/plugin/filter_suppress.rb

```diff
@@ -7,6 +8,7 @@ class SuppressFilter < Fluent::Plugin::Filter
     config_param :attr_keys,     :string,  default: nil
     config_param :num,           :integer, default: 3
     config_param :interval,      :integer, default: 300
+    config_param :give_feedback, :string,  default: nil
```

Comment:
`give_feedback` type should be a bool.
https://docs.fluentd.org/v1.0/articles/api-config-types#:bool

If it is a bool, `option_give_feedback?` method is not necessary.

Reply:
Thanks, @fujiwara . Thanks for the pointer to the docs. If it's not possible for the value to become nil, then sure, `option_give_feedback?` isn't needed.